src/leveldbwrapper.h

```diff
@@ -168,6 +190,19 @@ class CLevelDBWrapper
     {
         return pdb->NewIterator(iteroptions);
     }
+
+    /**
+     * Return true if the database managed by this class contains no entries.
+     */
+    bool IsEmpty()
+    {
+        leveldb::Iterator* it = NewIterator();
```

Comment:
Yeah, looks like this [only comes allocated](https://github.com/google/leveldb/blob/master/include/leveldb/table.h#L48).